src/natcap/invest/sdr/sdr.py

```diff
@@ -562,13 +623,39 @@ def execute(args):
         target_path_list=aligned_list,
         task_name='align input rasters')
 
+    mutual_mask_task = task_graph.add_task(
+        func=pygeoprocessing.raster_map,
+        kwargs={
+            'op': lambda *x: 1,  # any valid pixel gets a value of 1
+            'rasters': aligned_list,
+            'target_path': f_reg['mask_path'],
+            'target_nodata': 0,
+        },
+        target_path_list=[f_reg['mask_path']],
+        dependent_task_list=[align_task],
+        task_name='create mask')
+
+    mask_tasks = {}  # use a dict so we can put these in a loop
+    for key, aligned_path, masked_path in zip(input_raster_key_list,
+                                            aligned_list, masked_list):
+        mask_tasks[f"masked_{key}"] = task_graph.add_task(
+            func=pygeoprocessing.raster_map,
+            kwargs={
+                'op': lambda array, mask: array,
```

Comment:
Definitely!  Masking is such an easy thing to express within the context of `raster_map`, it was just too good to pass up.

Reply:
Unfortunately, using a `lambda` like this will break when `n_workers > 0`. I just discovered this issue while working on https://github.com/natcap/invest/pull/1437. Because lambdas can't be pickled, they can't be directly passed into a taskgraph Task. `op` has to be a named function defined at the top level of the module.

Reply:
Aha, of course!  Thanks for the reminder.  Patched.

Reply:
@phargogh should this be `_mask_op` now in `kwargs`?

Reply:
Oh yes, I missed one lambda and patched it in f33cf7098